packages/trader/src/sass/app/modules/mt5/mt5-dashboard.scss

```diff
@@ -1026,17 +1027,28 @@
             flex-direction: column;
             justify-content: space-around;
             align-items: center;
-            height: 24rem;
+            height: auto;
             padding: 0 3rem;
+            overflow-y: scroll;
```

Comment:
I am not sure if it's a good idea to have overflow-y for message content could you please check it with QA team

Reply:
Sure, will clarify  it with azadeh  and provide a feedback as soon as possible

Reply:
@mehrzad-fs  discussed with azadeh and she texted it's okay, if all other good, I think it's good idea send all of this to QA and they will check

Reply:
@mikita-deriv Please use `ThemedScrollbars` to make the content scrollable. We use `ThemedScrollbars` to make sure scrollbars look the same across different browsers.

Reply:
Sure @balakrishna-binary